### ISSUE TITLE
invalid point updates fail silently

### ISSUE THREAD

**Issue by michielbdejong** [tosdr/phoenix#355]
I tried to update the service of https://edit.tosdr.org/points/2297 from '(NONE)' to 'Google', but nothing changed when I did that.

**Comment by michielbdejong** [comment]
```sh
update!
{"title"=>"Processes a personal information (email, id but also device info, location)", "source"=>"https://www.google.be/intl/en/policies/privacy/", "status"=>"approved", "rating"=>"", "analysis"=>"f calls, Internet protocol address, device event information such as crashes, system activity, hardware settings, browser type, browser language, the date and time of your request and referral URL, cookies that may uniquely identify your browser or your Google Account\".\r\n\r\nCollects location, Unique application numbers, local storage, cookies.\r\n\r\nThose information can also be associated with information collected by Youtube (Google) partners.\r\n\r\n\r\n", "topic_id"=>"41", "service_id"=>"274", "point_change"=>""}
   (0.1ms)  BEGIN
  Service Load (0.3ms)  SELECT  "services".* FROM "services" WHERE "services"."id" = $1 LIMIT $2  [["id", 274], ["LIMIT", 1]]
  Topic Load (0.3ms)  SELECT  "topics".* FROM "topics" WHERE "topics"."id" = $1 LIMIT $2  [["id", 41], ["LIMIT", 1]]
   (0.2ms)  ROLLBACK
{:rating=>["can't be blank", "is not a number"]}
Redirected to http://localhost:3000/points/1118

**Comment by michielbdejong** [comment]
yes, so it was ok that the edit didn't stick, but the problem is that the user cannot know that choosing a rating is necessary, nor that their edit was discarded. we need to do this verification in the view somehow.

**Comment by madoleary** [comment]
Fixed. See #358

**Comment by Vinnl** [comment]
I'm assuming this can be closed now that #358 is merged? :)
